src/core/helpers.mjs

```diff
@@ -170,3 +170,10 @@ export function isNumeric (str) {
   return !isNaN(str) && // use type coercion to parse the _entirety_ of the string (`parseFloat` alone does not do this)...
     !isNaN(parseFloat(str)) // ...and ensure strings of whitespace fail
 }
+
+export function validatePath (input) {
```

Comment:
Need a doc block to explain what this function does.

Reply:
Since you are out, I'll merge this and create a small PR to fix it.

Reply:
Since you are out, I'll merge this and create a small PR to fix it.